scripts/automation/commandlint/run.py

```diff
@@ -0,0 +1,74 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+import pkgutil
+
+import argparse
+import os
+import sys
+
+from automation.utilities.path import filter_user_selected_modules_with_tests
+from azure.cli.core.application import APPLICATION, Application
+from azure.cli.core.application import Configuration
+from azure.cli.core.commands import load_params, _update_command_definitions
+
+from importlib import import_module
+
+
+def dump_no_help(modules):
+    cmd_table = APPLICATION.configuration.get_command_table()
+    exit = 0
+    for cmd in cmd_table:
+        cmd_table[cmd].load_arguments()
+
+    for mod in modules:
+        # print('loading params for', mod)
+        try:
+            import_module('azure.cli.command_modules.' + mod).load_params(mod)
+        except Exception as ex:
+            print("EXCPETION: " + ex.message)
+
+    _update_command_definitions(cmd_table)
+
+    for cmd in cmd_table:
+        descrip = cmd_table[cmd].description
+        if descrip is None or descrip is "":
+            exit = 1
+            print(cmd)
+
+        for key in cmd_table[cmd].arguments:
+            if cmd_table[cmd].arguments[key].type.settings.get('help') is None:
+                exit = 1
+                print(cmd + " " + str(cmd_table[cmd].arguments[key].name))
+    sys.exit(exit)
```

Comment:
Suggest return a value here. return 1 for error, 0 for ok.
Then in __main__ below, exit appropriately instead of exiting in this method.